tasks/src/image-to-text/data.ts

```diff
@@ -0,0 +1,58 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from 12M image-text of Reddit",
+			id:          "red_caps",
+		},
+        {
+			// TODO write proper description
+			description: "Dataset from images generated by Stable diffusion on Discord",
+			id:          "poloclub/diffusiondb",
+		},
+
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "savanna.jpg",
```

Comment:
Can you fix the path of the image itself? it should be `image-to-text/savanna.jpg` to work, not `image-text/savanna.jpg`.